tomography_preprocessing/denoising/care4relion/train_neural_network.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+import pandas as pd
+import starfile
+import rich
+import typer
+import subprocess
+from rich.progress import track
+
+from ._utils import *
```

Comment:
avoid this like the plague! It's impossible to know what came from where and which names might have been clobbered

Reply:
was just laziness from me soz! fixed